src/index.html

```diff
@@ -7,6 +7,381 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="header__nav">
```

Comment:
<img width="995" alt="image" src="https://user-images.githubusercontent.com/99403198/216768247-323d833c-0196-45e5-9ff3-2c106c6bc406.png">

```suggestion
      </a>
      
      <nav class="header__nav">
```